### DISCUSSION TITLE
Inject dictionary of bot instances

### DISCUSSION THREAD

**Discussion by atassis** [bukhalo/nestjs-telegraf#872] [Q&A]
I am setting multiple instances dynamically and want to inject all inctances by `Record<BotNameString, BotInstance>`. Can you implement that or share any ideas how can I make by myself with a PR?

**Comment by evilsprut** [comment]
Something like `@InjectAllBots()`?

**Reply by atassis** [reply]
Something like that, yep
But why did you make a discussion? Issues are about features also

**Comment by evilsprut** [comment]
Done in https://github.com/hypeertech/nestjs-telegraf/pull/875 Thx, @atassis
